### PR TITLE
lesson2-1

### PR DESCRIPTION
**Pull request by lucascheadle**

### CODE REVIEW

**Comment by lucascheadle** [comment]
Lesson03 exercise01 was put here by mistake.

**Comment by lucascheadle** [comment]
In error I initiated a new .git in my new Lesson03 dir. I realized we're just adding to the first .git we created upon making our  "html200-winter2020". Just add/commit that new Lesson03 dir to it. yes?

**Comment by lucascheadle** [comment]
Also, in Lesson03 ex1 I don't understand why the join wouldn't work without establishing a new variable separate from arr?

File: Lesson02/claculator.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+let y = 20;
+console.log(x * y);
+
+let z = 15;
+let xx = 25;
+console.log(z * xx);
+
+console.log((xx + x) / z);
+
+console.log((z - xx) + y / x);
```
Comment (jimmynono) [inline Lesson02/claculator.js:11]: Looks good. Only change I'd make is group the code like below.

```
let x = 10;
let y = 20;

console.log(x * y);

let z = 15;
let xx = 25;

console.log(z * xx);
console.log((xx + x) / z);
console.log((z - xx) + y / x);
```

File: Lesson02/canDrive.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+let firstName = "Bob";
+let lastName = "Rogers";
+let age = 22;
+let limit = 16;
+let x = (age - limit);
+if (age >= limit) {
```
Comment (jimmynono) [inline Lesson02/canDrive.js:6]: you'll want a space before this.

**Pull request closed without merging**